html_css/v2/main-course/browser-compatibility.md

```diff
@@ -0,0 +1,70 @@
+### Introduction
+As you continue your web development journey, it is important to keep in mind that the end users of your work might be using a variety of browsers: Chrome, Internet Explorer, Firefox, and Safari to name a few. At the same time, the number of users using mobile operating systems is growing rapidly, therefore you should also consider the mobile versions of different browsers.
+
+### Learning Outcomes
+By the end of this lesson, you should:
+
+- Understand browser compatibility and its history.
+- Know how new CSS features make their way into browsers.
+- Know how to check for compatibility.
+
+### Browser History
+The history of modern browsing began back in December of 1990 with the release of WorldWideWeb browser. It was written by Tim Berners-Lee while working for the European nuclear research agency known as CERN. It was later renamed to Nexus, to avoid confusion with the World Wide Web. 
+
+Nexus was the first of its kind, and allowed users to view basic style sheets, read newsgroups and even had spellcheck! It might not seem like a lot today, but at that time it was truly groundbreaking. 
```

Comment:
```suggestion
Nexus was the first of its kind, and allowed users to view basic style sheets, read newsgroups, and even had spellcheck! It might not seem like a lot today, but at that time it was truly groundbreaking. 
```